core/src/apps/bitcoin/get_public_key.py

```diff
@@ -21,7 +21,13 @@ async def get_public_key(ctx: wire.Context, msg: GetPublicKey) -> PublicKey:
     node = keychain.derive(msg.address_n)
 
     if (
-        script_type in [InputScriptType.SPENDADDRESS, InputScriptType.SPENDMULTISIG]
+        script_type
```

Comment:
This should be something like this:

`if script_type in [InputScriptType.SPENDADDRESS, InputScriptType.SPENDMULTISIG] or coin.decred and script_type in [InputScriptType.SPENDSSRTX, InputScriptType.SPENDSSGEN]`